### PR TITLE
[Microsoft MPI] make dlls executable

### PR DESCRIPTION
**Pull request by simonbyrne**

### CODE REVIEW

File: M/MicrosoftMPI/build_tarballs.jl [hunk 1]
```diff
@@ -28,6 +28,7 @@ fi
 cd ${WORKSPACE}/destdir/
 
 chmod +x *.exe
+chmod +x *.dll
```
Comment (giordano) [inline M/MicrosoftMPI/build_tarballs.jl:31]: No need to, now do this automatically: https://github.com/JuliaPackaging/BinaryBuilder.jl/pull/992

Comment (giordano) [reply M/MicrosoftMPI/build_tarballs.jl:31]: Just do a dummy edit to trigger registration

Comment (simonbyrne) [reply M/MicrosoftMPI/build_tarballs.jl]: done!

**Comment by giordano** [comment]
I confirm all files in the `bin/` directory of the [generated tarball](https://julia-bb-buildcache.s3.amazonaws.com/c50d9e459744a8e3decb47e42201ad232a1b78fb70f36b37cf8a3eb25efeb0b3/6f7f61ad4bd5886de445dc34a76b0e05db1243ed623139349c4f6d2e329d0670/x86_64-w64-mingw32.tar.gz) are automatically executable.

**Pull request merged**